saleor/settings.py

```diff
@@ -419,3 +419,6 @@ def get_host():
     'a': ['href', 'title'],
     'img': ['src']}
 ALLOWED_STYLES = ['text-align']
+
+GENERATED_SEO_MAX_LENGTH = 155
```

Comment:
We should totally drop `GENERATED_SEO_MAX_LENGTH` and do a `description_text[:300]` since we are also dropping the suffix and setting a max length of 300 on the field then?

Reply:
While using `ModelForms`, Django inherits  `max_length` attribute directly from the models.
I'd drop both constants, as this can be handled by max_length param on the model. 

When it comes to trimming descriptions, I think we should trim it after a full word, as close to 300 chars as possible.

Reply:
I implemented a trimming that doesn't break words at https://github.com/mirumee/saleor/pull/1899/commits/455aedd46884ddb949c52f19a2b97d8f38c321b4. 

I tried no to add a heavy text dependency to allow unicode languages (ex. Japanese, Korean, ... since they have different whitespace and punctuation characters).

My solution was to the regex token `\p{P}` to match punctuation in a unicode basis. Only issue: it not supported by python. So the dependency `regex` was added, it's a popular replacement for the `re` built-in module.

Also, the reason why I didn't make a regex that would look for any char that is note a character and set the unicode flag and not add a dependency, is that because it doesn't match every languages, despite the unicode flag.